ajax/libs/Swiper/package.json

```
@@ -16,9 +16,9 @@
   "npmName": "swiper",
   "npmFileMap": [
     {
```

not entirely sure but I think we should keep the `basePath`. 
```suggestion
    {
      "basePath": "",
```

---

Emm.... althought it has a little weird, I have commited it please check again~